dtests/src/test/java/io/snappydata/hydra/misc/MiscTest.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2018 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+package io.snappydata.hydra.misc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import io.snappydata.hydra.cluster.SnappyTest;
+import util.TestException;
+
+public class MiscTest extends SnappyTest {
+  protected static MiscTest miscTestInst;
+  int colCnt = 6;
```

Comment:
Remove the hardcoded value, instead make it read the value dynamically depending on the table schema.